features/google/src/components/awakeningLocationMinistory/awakeningLocationMinistory.jsx

```diff
@@ -0,0 +1,135 @@
+import "./awakeningLocationMinistory.css";
+import React, { useState } from "react";
+import SourceInfoButton from "../sourceInfoButton/sourceInfoButton.jsx";
+import i18n from "!silly-i18n";
+
+export const AwakeningLocationSummary = ({ dateData }) => {
+    const defaultDate = Object.entries(dateData).find(
+        (entry) => !!entry[1].firstActivity
+    )[0];
+    const [selectedDate, setSelectedDate] = useState(defaultDate);
+    const data = dateData[selectedDate];
+    const dateSelection = false;
+    const handleChangeDate = ({ target }) => {
```

Comment:
Not entirely sure what you're doing here, but looks a bit ad hoc and. Does the new library you're using have something to help with this?